internal/repos/store.go

```diff
@@ -102,7 +101,7 @@ type store struct {
 	*basestore.Store
 
 	// Logger used by the store. Defaults to log15.Root().
-	Log logging.ErrorLogger
+	Log log.Logger
```

Comment:
```suggestion
	Logger log.Logger
```

☝️ is the general convention that is propagating through the codebase

Reply:
Fixed 🙇🏼